src/sonic-yang-models/yang-models/sonic-events-swss.yang

```diff
@@ -38,11 +38,11 @@ module sonic-events-swss {
                 The name of the flapping i/f and status are the only params.";
 
             leaf ifname {
-                type leafref {
-                    path "/port:sonic-port/port:PORT/port:PORT_LIST/port:name";
+                type string {
+                    pattern 'Ethernet[0-9]{1,2}';
```

Comment:
>Ethernet [](http://example.com/codeflow?start=29&length=8)

There is use case like `Ethernet124` not covered by this pattern.

Reply:
Changed